File: android/testData/lint/global/genBackupDescriptor/MainActivity.java
package p1.pkg;

import android.app.Activity;
import android.content.SharedPreferences;
import android.os.Bundle;

public class MainActivity extends Activity {

  private static final String PREF_NAME = "p1.pkg.PREFS";
  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    SharedPreferences prefs = getSharedPreferences(PREF_NAME, MODE_PRIVATE);

    SharedPreferences prefs2 = getSharedPreferences(getString(R.string.pref_name), MODE_PRIVATE);
  }
}

File: android/integration/testData/applychanges/src/main/java/com/example/applychanges/MainActivity.kt
package com.example.applychanges

import android.app.Activity
import android.os.Bundle
import android.util.Log
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val label = TextView(this)
        label.setText("Hello Minimal World!")
        Log.i("MainActivity", "Hello Minimal World!")
        setContentView(label)
    }

    override fun onResume() {
      super.onResume()
      // EASILY SEARCHABLE ONRESUME LINE (this comment exists for the test to be able to find/replace from here until the next comment)
      printBefore()
      // END ONRESUME SEARCH
    }

    fun printBefore() {
      Log.i("MainActivity", "OnResume Before with resource status: " + getString(R.string.status))
    }

    fun printAfter() {
      Log.i("MainActivity", "OnResume After with resource status: " + getString(R.string.status))
    }
}
